modules/tutorials/partials/student-mvn-directory-structure.adoc

```diff
@@ -0,0 +1,48 @@
+// # tag::basic[]
```

Comment:
Just wanted to acknowledge that this is a brilliant way of illustrating a directory structure! 👏

Reply:
Thanks :-)

I think nicked the idea from Sarah at OpenDevise. She does something similar for the Asciidoctor documentation.